src/components/SelectCol.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+class SelectCol extends React.Component {
+    constructor(props) {
+        super(props);
+        this.getCheckedValues = this.getCheckedValues.bind(this);
+        this.state = {
+            languages: [ "Arabic", "Serbian", "Croatian", "Russian", "German", "Hebrew", "French",
+                "Hungarian", "Slovak", "Spanish", "Portugues", "Turkce", "Greek", "Romanian"],
+            defaultLans: this.props.default
+        }
+    }
+
+    getCheckedValues(index){ //Get the selected Value and pass to other component
+        //toggle the default languages
+        if (index == null) {
+        }else {
+            const newLans = this.state.defaultLans.slice() //copy the array
+            newLans[index].value = !this.state.defaultLans[index].value;  //execute the manipulations
+            this.setState({ defaultLans: newLans }) //set the new state
+        }
+
+        const values = [];
+        const boxes = document.getElementsByClassName("custom-control-input");
+        for(let i=0; i< boxes.length; i++){
+            if(boxes[i].checked){
+               values.push(boxes[i].id);
+            }
+        }
+
+        //pass the values to the searchPage component
+        this.props.parentCallback(values);
+    }
+
+    //handle Event functions
+    openForm(){
+        const x = document.getElementById("language-options");
+        if (x.style.display === "none") {
+            x.style.display = "block";
+        } else {
+            x.style.display = "none";
+        }
+
+        document.addEventListener('click',function(e){
+            x.style.display = "none";
+        });
+    }
+
+    handleClick(e){ //stop propagation of the react
+        e.stopPropagation();
+        e.nativeEvent.stopImmediatePropagation();
+    }
+
+    render() {
+        console.log( "The Select Col render function gets called" );
+
+        //Style will move to css file later
+        const buttonStyle ={
```

Comment:
remember to move this part to a css file before merging

Reply:
I thought about this. However, I found this part the CSS is not too much. So I decided not to create a separate CSS file

Reply:
ok, that's fine then. should remove the comments in this case :)

Reply:
ok. I forgot... I wrote this in the first commit.